cmd/consensus/main.go

```diff
@@ -695,12 +668,45 @@ func loadDKGPrivateData(dir string, myID flow.Identifier) (*dkg.DKGParticipantPr
 	return &priv, nil
 }
 
-func loadMachineAccountPrivateData(node *cmd.FlowNodeBuilder) (*bootstrap.NodeMachineAccountInfo, error) {
-	data, err := io.ReadFile(filepath.Join(node.BaseConfig.BootstrapDir, fmt.Sprintf(bootstrap.PathNodeMachineAccountInfoPriv, node.Me.NodeID())))
+func createDKGContractClient(node *cmd.FlowNodeBuilder, accessAddress string) (module.DKGContractClient, error) {
```

Comment:
Regarding testing this, I think we can test at the level of these methods which decide which client object to produce. 

We should check that:
* with the machine account file written to disk and `access-address` flag set, we get a real client instance
* with either machine account file omitted or `access-address` omitted, we get a fake client instance